packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -43,13 +43,13 @@ const main = () => {
             ``,
             `Usage inside NPM scripts:`,
             ``,
-            `  The PWA Kit CLI is used in NPM scripts so you can conveniently`,
+            `  The PWA Kit Dev is used in NPM scripts so you can conveniently`,
```

Comment:
As a follow up to @adamraya's philosophical reflections, should we call this "PWA Kit Developer Tools" or something?

Reply:
Haha. I can buy into that.

Reply:
Lets go with PWA Kit Developer Tools. We can always change it in the future. It's just copy.

Reply:
I was secretly thinking the same, but I didn't want to be the only one pushing for that name :p